pkg/agent/datamodel/sig_config.go

```diff
@@ -447,6 +452,20 @@ var (
 		Version:       LinuxSIGImageVersion,
 	}
 
+	SIGUbuntuMinimalContainerd2204ImageConfigTemplate = SigImageConfigTemplate{
+		ResourceGroup: AKSUbuntuResourceGroup,
+		Gallery:       AKSUbuntuGalleryName,
+		Definition:    "2204minimalcontainerd",
```

Comment:
this reminds me, did you make sure to update the set SKU_NAME logic in the vhd builds?

Reply:
I did :) https://github.com/Azure/AgentBaker/pull/3370/files#r1254790224

Reply:
noice I missed that lol